generic3g/actions/VerticalRegridAction.F90

```diff
@@ -87,36 +101,65 @@ subroutine update(this, importState, exportState, clock, rc)
       real(ESMF_KIND_R4), pointer :: x_in(:,:,:)
       real(ESMF_KIND_R4), pointer :: x_out(:,:,:)
 
-      real(ESMF_KIND_R4), pointer :: v_in(:,:,:)
-      real(ESMF_KIND_R4), pointer :: v_out(:,:,:)
+      real(ESMF_KIND_R4), pointer :: v_in(:)
+      real(ESMF_KIND_R4), pointer :: v_out(:)
 
-      integer :: i, j, k
-      integer, parameter :: IM = 2, JM = 2, LM = 2
+      integer :: istart, iend, jstart, jend, i, j
 
-      if (associated(this%v_in_coupler)) then
-         call this%v_in_coupler%run(phase_idx=GENERIC_COUPLER_UPDATE, _RC)
-      end if
+      ! if (associated(this%v_in_coupler)) then
+      !    call this%v_in_coupler%run(phase_idx=GENERIC_COUPLER_UPDATE, _RC)
+      ! end if
 
-      if (associated(this%v_out_coupler)) then
-         call this%v_out_coupler%run(phase_idx=GENERIC_COUPLER_UPDATE, _RC)
-      end if
+      ! if (associated(this%v_out_coupler)) then
+      !    call this%v_out_coupler%run(phase_idx=GENERIC_COUPLER_UPDATE, _RC)
+      ! end if
```

Comment:
I think these calls are essential - please explain why they are commented out.

Reply:
With the couplers not associated, these are not getting exercised at the moment. Once I have the Model -> FixedLevels regridding active, and as I understand the role these couplers play, I'll turn them back on.